simulator/src/main/java/com/hazelcast/simulator/utils/CloudProviderUtils.java

```diff
@@ -59,4 +63,8 @@ public static boolean isStatic(String cloudProvider) {
     public static boolean isEC2(String cloudProvider) {
         return PROVIDER_EC2.equals(cloudProvider);
     }
+
+    public static boolean isSupported(String cloudProvider) {
```

Comment:
isnt this misleading? simulator itself supports an arbitrary cloud, it's just a matter of adding a provisioning shell script, isnt it?

Reply:
I see your point but I'm not sure about the resolution. The whole `CloudProvider` concept is only relevant for Simulator in the context of `provisioner` and in that case, `provisioner` really doesn't support out of the box other clouds. You can of course implement the shell script but that's effectively implementing the support for that cloud provider.

So I would say it's debatable whether it should or shouldn't be named `isSupported`. I'm trying to think about it but no better name comes up. 

If you don't mind, I'll go ahead and merge the PR. If you come up with a better name, please comment and I'm happy to rename the method in a separate PR.